cfonts/cli.py

```diff
@@ -127,6 +127,13 @@ def parse_args() -> argparse.Namespace:
         help="Set this option to generate your own gradients. Each color set "
         "in the gradient option will then be transitioned to directly.",
     )
+    parser.add_argument(
+        "-u",
+        "--copyable",
```

Comment:
The option name doesn't make much sense to me. The rendered text is also copyable. I will suggest `--raw` or similar.